### PR TITLE
Fix offsets for iOS 14

### PR DESCRIPTION
**Pull request by luizmb**

### CODE REVIEW

**Review by melle — APPROVED** [review]

File: Sources/UIExtensions/Components/SwiftUI/UIKit Wrappers/PagingScrollView.swift [hunk 1]
```diff
@@ -72,25 +74,18 @@ public struct PagingScrollView<DataCollection: RandomAccessCollection, PageType:
...
                         .frame(width: self.tileWidth(for: outerProxy))
                 }
             }
-            .onAppear {
-                self.currentScrollOffset = self.offsetForPageIndex(self.activePageIndex, proxy: outerProxy)
-            }
             .offset(x: self.stackOffset(for: outerProxy), y: 0)
             .background(Color.black.opacity(0.00001)) // hack - this allows gesture recognizing even when background is transparent
             .frame(width: self.contentWidth(for: outerProxy))
-            .onReceive(NotificationCenter.default.publisher(for: UIDevice.orientationDidChangeNotification), perform: { (_) in
```
Comment (zweigraf) [inline Sources/UIExtensions/Components/SwiftUI/UIKit Wrappers/PagingScrollView.swift:85]: This orientation change notification is not necessary anymore? This was to fix a bug, I think

Comment (luizmb) [reply Sources/UIExtensions/Components/SwiftUI/UIKit Wrappers/PagingScrollView.swift:85]: currentScrollOffset as a State wasn't being updated in several situations. As a calculated property now, it doesn't get stale any more, as long as the other properties (GeometryReader) is updated.
I can test specially that in The App, but even other problems like driving current page (scrolling position) from page control gesture that was broken, now it's fixed.

Comment (zweigraf) [reply Sources/UIExtensions/Components/SwiftUI/UIKit Wrappers/PagingScrollView.swift:85]: Yeah not sure anymore. If this still triggers a bug, we could reset `dragOffset` here on this notification.

**Review by zweigraf — APPROVED** [review]

File: Sources/UIExtensions/Components/SwiftUI/UIKit Wrappers/PagingScrollView.swift (hunk 1 above)
Comment (luizmb) [reply Sources/UIExtensions/Components/SwiftUI/UIKit Wrappers/PagingScrollView.swift:85]: Will need to merge this to test it better, to be honest. Adding UIExtensions as local SPM was pretty chaotic as its dependency goes through TeufelCommon, another SPM dependency. In the end I had to add this struct with a different name to test, it fixed (although I still don't fully understand the logic behind that offset and why it's different between iOS 13 and 14). So I'm pretty sure more PRs here will be necessary to fix completely that bug.

**Pull request merged**